assets/src/edit-story/app/media/media3p/reducer.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { shallowEqual } from 'react-pure-render';
+
+/**
+ * Internal dependencies
+ */
+import providerReducer from './providerReducer.js';
+
+/**
+ * External dependencies
```

Comment:
@cvolzke4 Can you please fix the mixed up comments in this file?

Reply:
oops, thanks for spotting that. Fixed: https://github.com/google/web-stories-wp/pull/2806.